packages/aws-cdk-lib/aws-lambda-event-sources/test/s3.test.ts

```diff
@@ -80,4 +80,78 @@ describe('S3EventSource', () => {
       },
     });
   });
+
+  test('test S3EventSourceV2', () => {
```

Comment:
Can we add another test with `S3EventSourceV2` using a `Bucket` resource instead of a `IBucket` resource? I would expect it to also work but want to add a test to make sure.